### PR TITLE
Re-enable ruby add to your app tests

### PR DESCRIPTION
**Pull request by gkaemmer**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -348,6 +348,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 3.0
```
Comment (gj) [inline .github/workflows/test.yml:350]: Any idea why we're using Ruby 3 in the `docs_test` job but Ruby 2.4 in the `ruby_test` job?

Comment (gkaemmer) [reply .github/workflows/test.yml:350]: No idea. Seems not good. Will try downgrading here and see what happens. If it doesn't work, I may punt on it though

Comment (gkaemmer) [reply .github/workflows/test.yml]: Seems to work

File: docs/examples/add-to-your-application/Makefile [hunk 2]
```diff
@@ -14,5 +18,5 @@ test-node:
 test-ruby:
 	# TODO disabled because these won't pass CI for some reason
```
Comment (dhatch) [inline docs/examples/add-to-your-application/Makefile:19]: Remove comment.

**Review by dhatch — APPROVED** [review]
lgtm besides comment

**Pull request merged**